Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM alpine:3.12
```

Comment:
We use distroless everywhere, is there reasoning for using alpine here? From a conformance standpoint would be better to use one base image?

Reply:
I agree, I'll fix that :)